controller/src/schemas/base.py

```diff
@@ -216,53 +147,37 @@ def __str__(self):
 
 class BaseWithMetadata(Base):
     name: str
+    id: Optional[str] = None
```

Comment:
`id` is a builtin in Python. Please rename to `id_` or `uid` or if you have a better name, go for it.